src/Apps/CollectorProfile/Routes/Saves/Components/CreateNewListModal/CreateNewListModal.tsx

```diff
@@ -82,14 +90,16 @@ export const CreateNewListModal: React.FC<CreateNewListModalProps> = ({
       })
 
       const response = createCollection?.responseOrError
-      const artworkListId = response?.collection?.internalID!
+      const artworkListId = response?.collection?.internalID
 
-      onComplete({
-        internalID: artworkListId,
-        name: values.name,
-      })
+      if (artworkListId) {
+        onComplete({
+          internalID: artworkListId,
+          name: values.name,
+        })
 
-      trackAnalyticEvent(artworkListId)
+        trackAnalyticEvent(artworkListId)
```

Comment:
#superminor - i know this was here before but it might be good to make the function name a bit more descriptive and tied to the type that its associated with